### ISSUE TITLE
Error message in product review form not being translated

### ISSUE THREAD

**Issue by Kivike** [magento/magento2#10474]
### Preconditions

1. Magento 2.1.7 CE

### Steps to reproduce

1. Submit review without setting rating

### Expected result

1. Error 'Please select one of each of the ratings above.' should be translated

### Actual result

1. Error is always in english

**Comment by emmathepossum** [comment]
The problem is that the error message is hardcoded into the template file:
 [`module-review/view/frontend/templates/form.phtml` on line 40](https://github.com/magento/magento2/blob/2.1.7/app/code/Magento/Review/view/frontend/templates/form.phtml#L40).


Until this is fixed, you can do the following:

1. Override the template file (`Magento_Review/templates/form.phtml` in your theme) and change Line 40 to something like this: `data-validate="{'required-rating':true}"`

2. Override the validation.js (`web/js/validation.js` in your theme) and add this to the rules object (Line 167):
```js
'required-rating': [
    function(value) {
        return value !== undefined;
    },
    $.mage.__('Please select one of each of the ratings above.')
]
```

3. Add the overridden validation.js to your `requirejs-config.js`
```
var config = {
    map: {
        '*': {
            'mage/validation': 'js/validation',
            'mage/validation/validation': 'js/validation'
        }
    },
    "deps": [
        'mage/validation'
    ]
};
```

4. Translate the phrase in your csv file

5. Clear caches

**Comment by Echron** [comment]
I will see if i can fix this one

**Comment by okorshenko** [comment]
@Kivike thank you for your report. The issue is already fixed in 2.2-develop branch (PR: #11069 by @Echron) and will be available in the Magento 2.2.2 release

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-80224

**Comment by okorshenko** [comment]
Hi @Kivike. Thank you for your report. 
The issue has been fixed in magento-engcom/magento2ce#1323 by @magento-engcom-team in 2.3-develop branch
Related commit(s):
 - 809ed03393aeb55d51897c88561d9cdf1d0907a3
 - de9d15a247e165baef6f8138713077d80ac798ce
 - b322c098e8d39bfb2ef8a4287df7a5cfba4b2a84
 - 703d9e0a5d3367b6719b4445957438e066f497ed
 - ecfe9010ba5a0fc67e9c8a1fccbf202695d4ec3d
 - a90dbca28e49bf87b219bb9bcb091fb5a964e265
 - b4a7213a7efcd532d58bb600ad483709cc1e970d
 - 660c62a82a041c3f203c9e395e2bc62e7788940f
 - 8d48fd346467aa2267a97fc838d42c33a7399138
 - d0b525b16ba86b3bd59bb8e23adf3d9687de16df
 - 0dd84cd92c5c2d66471e3a4cef7768da33afdaf6
 - 6746da4c6b1b535043b79f7ccd5af91488008d34
 - 413f9394c659ca24dd942a23a908c8500088436e
 - 2eca1d0038d9ecf33c33f4bd9cff28ff4e20ceb3
 - f9976b9365b8d445f932274eb2160127918b888c
 - 4c77e4f5ebb0f7d82efb24a0f02ab7ef7773d8a5
 - 6c62727ca10389745b92ac683e5bb8deee7496f3
 - d8915032a9b00b93601f5dd8316272a5949b94f2
 - d10f55e4016b91dacb2991b44b5a65fbd5ccd7ab
 - e2ff943b8011aa6604e7280e9581f074ebd5ecb3
 - e27445da1e3039d898c53a3f8a9052094a892038
 - f0711edabca0ecc5daf384a838576822a6f4d4af
 - 79de0d6ad14e04ebcc645f6e336082cde6035423
 - a30789d9ec4bc536423b350509790cfc018c5691

The fix will be available with the upcoming 2.3.0 release.

**Comment by sadon** [comment]
Magento 2.3.0 

_Please select one of each of the ratings_

 doesn't translate
